README.md

```diff
@@ -93,6 +93,10 @@ A dashboard is provided to offer real-time statistics and historical graphs.  It
 
 - [vagrant-vmpooler](https://github.com/briancain/vagrant-vmpooler) Use Vagrant to create and manage your vmpooler instances.
 
+## Development and further documentation
+
+For more information about setting up a development instance of VM Pooler or other subjects, see the [docs/](docs) directory.
```

Comment:
> VM Pooler

Product is referred to as "vmpooler", not "VM Pooler" in the rest of the documentation.